src/View/ViewableData.php

```diff
@@ -426,9 +426,11 @@ public function castingHelper($field)
      *
      * @param string $field
      * @return string
+     * @deprecated 5.4.0 Will be removed without equivalent functionality to replace it.
      */
     public function castingClass($field)
```

Comment:
Nothing uses this, so no need to replace it. If we do need this in the future it can be added to the new `CastingService`.